ggml/src/ggml-cpu/ggml-cpu-quants.c

```diff
@@ -2696,10 +2876,10 @@ void ggml_vec_dot_q5_0_q8_0(int n, float * restrict s, size_t bs, const void * r
     }
 
     sumf = vaddvq_f32(sumv0) + vaddvq_f32(sumv1);
-#elif defined(__wasm_simd128__)
+#elif defined __wasm_simd128__
     v128_t sumv = wasm_f32x4_splat(0.0f);
 
-    uint32_t qh;
+    uint32_t qh_;
```

Comment:
Is this change useful?

Reply:
naming shadow, not related to the current PR but is thrown up on compilation

Reply:
So should it be in this PR if it is not related to the current PR?

Reply:
It's a common thing in llama.cpp, you should look at other PRs. Style fixes don't necessary need its own PR, just nit-picking things.

Reply:
It is bad that it is "common". Clean separation of logic changes and formatting fixes is better for understanding "what happens here".